File: Angular/EstefaniasJeans/src/app/admin/mercaderia/model/Mercaderia.ts
import { Categoria } from "src/app/model/Categoria";
import { MercaderiaFoto } from "./MercaderiaFoto";

export interface Mercaderia{
    Id_Mercaderia:string;
    Id_Categoria:string;
    Nombre:string;
    Descripcion:string;
    Stock:number;
    StockRestante:number;
    Precio: number;
    Estado: boolean;
    Categoria: Categoria;
    MercaderiaFoto: MercaderiaFoto[];
}

File: EstefaniasJeans/EstefaniasJeans/Data/Model/Mercaderia.cs
using System;
using System.Collections.Generic;

namespace EstefaniasJeans.Data.Model
{
  public class Mercaderia
  {
    public Guid Id_Mercaderia { get; set; }
    public Guid Id_Categoria { get; set; }
    public Guid? Id_Persona { get; set; }
    public string Nombre { get; set; }
    public string Descripcion { get; set; }
    public int Stock { get; set; }
    public int StockRestante { get; set; }
    public decimal Precio { get; set; }
    public bool Estado { get; set; }
    public DateTime FechaCreacion { get; set; }
    public Categoria Categoria { get; set; }
    public Persona Persona { get; set; }
    public ICollection<MercaderiaFoto> MercaderiaFoto { get; set; }
    public ICollection<PedidoDetalle> PedidoDetalle { get; set; }
  }
}
